### PR TITLE
Add prop to support custom optionTextStyles

### PR DESCRIPTION
**Pull request by kajeevan025**
Just added support to customise the option text styles.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -124,17 +124,18 @@ export default class ModalFilterPicker extends Component {
...
     const { key, label } = rowData
 
     let style = styles.optionStyle
-    let textStyle = styles.optionTextStyle
+    let textStyle = optionTextStyle||styles.optionTextStyle
 
     if (key === selectedOption) {
       style = styles.selectedOptionStyle
-      textStyle = styles.selectedOptionTextStyle
+      textStyle = optionTextStyle||styles.selectedOptionTextStyle
```
Comment (hiddentao) [inline src/index.js]: Would be better to have another prop called `selectedOptionTextStyle` rather than re-use the normal style.

**Comment by hiddentao** [comment]
@kajeevan025 Don't need this anymore?

**Comment by kajeevan025** [comment]
sorry by mistake i closed

**Comment by kajeevan025** [comment]
can you merge

**Comment by hiddentao** [comment]
Great, thanks

**Pull request merged**